### DISCUSSION TITLE
How to autosize a property's div ('mv-autosize')?

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#897] [Q&A]
Hi,

Is there a way to automaticly resize the div element on `mv-mode='edit'`, while applying a style to it (trought another property)?

[Codepen here](https://codepen.io/galinhalx/pen/RwMZbOj)

I tried to understand @LeaVerou [codepen](https://codepen.io/leaverou/pen/bGRmQKv), but I was not able to apply it to other thing than `input` element

Tried to use `mv-autosize` but can't find information about it: it seems to work only on input...which should be fine, but how to apply it to my case, where I modify property class, throught another property?

by the way, for information, I'm trying to apply it to all properties, and not only to one specific id

**Comment by DmitrySharabin** [comment]
If I recall correctly,  Mavo uses [Stretchy](https://stretchy.verou.me/) under the hood. You might find the answer to your question in the docs.

**Reply by GalinhaLX** [reply]
Thanks @DmitrySharabin 

Very interesting project

If I understand correctly, as it's already used by Mavo, I shoudn't have to call it again throught `<script>`
Anyway, I tried both (with and without), also with `Stretchy.resize();` as script

It still does'nt work :(

In fact it works as soon as I click on 'Editing'->'Edit'  on main bar:
styles will be applied, and resize is fine 

But my goal is to avoid this 'Edit' double click each time I modify something :)
